### ISSUE TITLE
Add APK to releases

### ISSUE THREAD

**Issue by lakshaysethi** [begeekmyfriend/yasea#848]
Currently it is not easy for some person who does not have access to a laptop and then was to do right away to download the APK and use the app from this code without Android studio

One way to solve this problem is to put the APK in the releases tab in GitHub, so that mobile phone users without the laptop can download the APK, install and use the app.

**Comment by begeekmyfriend** [comment]
Yes I agree. Unfortunately I am no longer engaged in Android development and I have no Android Studio IDE at hand. Here is the [yasea-apk](https://github.com/begeekmyfriend/yasea-apk) repo where there is only 2.6 version released. It can be downloaded and installed on your phone with most of the feature even thought it is not the latest. Hope it can help you with it.
